### ISSUE TITLE
[Bug]: Python3 package doesn't support setting install location

### ISSUE THREAD

**Issue by gmckeown** [microsoft/winget-pkgs#45194]
### Brief description of your issue

The `--location` parameter is ignored when installing python.python.3 package, even with `--scope machine`.

### Steps to reproduce

```
winget install --scope machine --location c:\apps\python3 python.python.3
```
As of today, that's using the 3.10.2150.0 version of the package.

### Expected behavior

Python should install to my specified location (c:\apps\python3).

### Actual behavior

This will still install to the default C:\Program Files\Python... location.

I believe this is due to the package-level setting of the following in Python.Python.3.installer.yaml:
```
InstallerSwitches:
  InstallLocation: DefaultCustomTargetDir=<INSTALLPATH>
```
According to [Python docs](https://docs.python.org/3/using/windows.html#installing-without-ui), this only sets the path shown in the installer.

Since there is a different parameter depending on whether you're doing a per-user or per-machine install, I tested with the following in each of the Installer entries under Installers:

Where InstallAllUsers=0:
```
InstallLocation: DefaultJustForMeTargetDir=<INSTALLPATH>
```

Where InstallAllUsers=1:
```
InstallLocation: DefaultAllUsersTargetDir=<INSTALLPATH>
```

For my use-case (x64 per-machine install) this seems to do the trick.

### Environment

```shell
Windows Package Manager v1.1.13405
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.1466
Package: Microsoft.DesktopAppInstaller v1.16.13405.0
```

**Comment by gmckeown** [comment]
Tagging submitters of some of the more recent Python 3 packages for comment:

@nrudakov @jedieaston @remyjette @vedantmgoyal2009

**Comment by jedieaston** [comment]
Thanks for bringing this up. Once all of the linked PRs have merged, you should be good to go!

**Comment by gmckeown** [comment]
Awesome, thanks for the quick turnaround!

**Comment by denelon** [comment]
@gmckeown It appears they are all merged. Can you confirm this is sufficient to close this issue?

**Comment by gmckeown** [comment]
@denelon From a purely selfish perspective, I'm only really interested in the system-wide 64-bit 3.10.2 install and that is now working. So if that's enough then please do close the issue :)

**Comment by iamdejan** [comment]
@kristjanvalur FYI, I just made a PR for Python 3.11.4 [here](https://github.com/microsoft/winget-pkgs/pull/112391).

**Comment by kristjanvalur** [comment]
> @kristjanvalur FYI, I just made a PR for Python 3.11.4 [here](https://github.com/microsoft/winget-pkgs/pull/112391).

Thanks.  Does this ensure that this gets included in future releases too?  It seems previous fixes only affect specific releases and new releases revert back to the old behaviour.

**Comment by iamdejan** [comment]
> Thanks. Does this ensure that this gets included in future releases too? It seems previous fixes only affect specific releases and new releases revert back to the old behaviour.

Depends on how the manifests are created. If the new manifests are upgraded with Komac / Winget Release (or even copied and pasted) from 3.11.4, the fixes will be included. However, there are Python 3.12 manifests (albeit in beta version) created before my fix is created. I believe in order to incorporate the fixes in 3.12, the fixes need to be included in those manifests as well.
